### PR TITLE
ADD: New event card

### PR DESCRIPTION
**Pull request by kherkatary**
### ADD: 
New Event Card component

#### Card structure
- size : small | large
- card content to be passed as props

### CODE REVIEW

**Review by codadept — APPROVED** [review]

**Comment by codadept** [comment]
image

- Increase the font size of register and details buttons a bit.,

- Also if the old EventCard Component won't be used again then delete that component and rename the NewCard Component to EventCard Component

**Review by codadept — CHANGES_REQUESTED** [review]
Checkout the comment for requested changes

**Comment by codadept** [comment]
Also properly display the Event Card component in the Home Page

File: src/Pages/Home/Home.jsx [hunk 1]
```diff
@@ -9,16 +9,63 @@ import {
...
+
+      <EventCard
+        size="small"
+        id={dummy[0].id}
+        content={dummy[0].content}
+        title1={dummy[0].title1}
+        title2={dummy[0].title2}
+        image={dummy[0].image}
+      />
+
```
Comment (JOSU9435) [inline src/Pages/Home/Home.jsx]: display cards using **map** function

File: src/Components/EventCard/Large.module.scss [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+      padding: 7% 10%;
+
+      overflow: hidden;
+      display: -webkit-box;
+      -webkit-box-orient: vertical;
+      -webkit-line-clamp: 17;
+
+      @media screen and (max-width: 500px) {
+        font-size: 4vmin;
```
Comment (JOSU9435) [inline src/Components/EventCard/Large.module.scss]: rem unit would be more appropriate for font sizes

**Review by JOSU9435 — CHANGES_REQUESTED** [review]
image

- proportionally decrease the border radius of the smaller card with respect to the large one
- reduce the size of the title a bit such that it fits in one line
- properly implement ellipses on content overflow
- refrain from using px as units replace all the px units in CSS files with relative units like rem, %, vh ...
-

File: src/Components/EventCard/EventCard.jsx [hunk 3]
```diff
@@ -1,42 +1,47 @@
...
+          </div>
+        </div>
+        <div className={styles.back}>
+          <div className={styles.inner}>
+            <div className={styles.backSide}>
+              <h1 className={styles.backHead}>{props.title1}</h1>
+              <p className={styles.para}>{props.content}</p>
+              <div className={styles.buttons}>
+                <button className={styles.register}>Register</button>
+                <button className={styles.details}>Details</button>
```
Comment (JOSU9435) [inline src/Components/EventCard/EventCard.jsx]: replace  `<button>` tag to `<Link>` tag

File: src/Components/EventCard/Large.module.scss
Comment (JOSU9435) [inline src/Components/EventCard/Large.module.scss:1]: replace px unit with equivalent relative unit

File: src/Components/EventCard/Small.module.scss
Comment (JOSU9435) [inline src/Components/EventCard/Small.module.scss:1]: replace px unit with equivalent relative unit

**Review by JOSU9435 — CHANGES_REQUESTED** [review]

**Review by JOSU9435 — APPROVED** [review]

**Review by JOSU9435 — APPROVED** [review]

**Pull request merged**